### ISSUE TITLE
[solved - Pause Game on Focus Loss] - strange freeze on switching window

### ISSUE THREAD

**Issue by JTD420** [Yimura/YimMenu#240]
**Describe the bug**
Once the dll is injected, all works as it should. But when you switch window for example ALT + TAB to say discord and then back to GTA the game is in a frozen/unresponsive state. I've found that spam pressing ESC tends to fix things but without doing so you can hear the game audio going on in the background, even footsteps if you walk but the display itself is still frozen from the state it was in just prior to switching windows.

**To Reproduce**
1. Load into game SP or MP and inject the dll. Have another window open such as chrome or discord that you can switch too.
2. Click on ALT+TAB to switch from the GTA process to discord/chrome etc.
3. Click on ALT++TAB to swithc back to the GTA process.
4. See error/issue described above and possible workaround.

**Expected behavior**
Near seamless switching as is expected from the game without cheats injected.

**Logs**

**Build**
Select which applies for you:
- [x] I used YimMenu unmodified
- [ ] I use my own version of YimMenu

**Additional context**
I'm not sure what the culprit is for this but I've used other internal P2C's that haven't caused this weird unresponsive type behaviour upon switching back and forward so it can definitely be avoided once we know whats causing it ofc

**Solution**
As answered below, the solution for the is to go into the game settings, go to Graphics and make sure to disable `Pause Game On Focus Loss`. 
If you are still experiencing issues make sure to change your screen type to Windowed Borderless from the same Graphics Settings window!

**Comment by Yimura** [comment]
This is just the game being bad and is unrelated to YimMenu.

**Comment by howdoiusekeyboard** [comment]
If you want instant alt tab experience, set screen type window borderless mode
Your issue could also be related to other things, drivers and OS, I have never faced this issue.

**Comment by Yimura** [comment]
In my experience the game works really badly if I don't use `Windowed Borderless` and disable this feature:
![image](https://user-images.githubusercontent.com/24669514/169786082-396fd151-b197-4e42-80a3-b875528de84b.png)

Disabling that feature will prevent the bug from occurring.

**Comment by JTD420** [comment]
> In my experience the game works really badly if I don't use `Windowed Borderless` and disable this feature: ![image](https://user-images.githubusercontent.com/24669514/169786082-396fd151-b197-4e42-80a3-b875528de84b.png)
> 
> Disabling that feature will prevent the bug from occurring.

This was exactly what I needed to do! I had purchased a new alt account for GTA:O while I do testing of the menu in-case of bans etc but in doing so it lost some of my display/gfx settings and `Pause Game On Focus Loss` was one of them!

I exclusively play the game in windowed borderless anyway so once I made that change everything was working perfect and seamlessly again just like the expected behaviour of the game prior to injecting!
